cypress/integration/components/Select.spec.js

```diff
@@ -136,6 +136,39 @@ describe("Select", () => {
     });
   });
 
+  describe.only("Multiselect: options live update", () => {
```

Comment:
I think we need to remove the only here.

Reply:
Thank you! Good catch.